src/main/java/io/connect/scylladb/ScyllaDbSinkTaskHelper.java

```diff
@@ -95,7 +95,9 @@ public BoundStatement getBoundStatementForRecord(SinkRecord record) {
       boundStatement.setDefaultTimestamp(topicConfigs.getTimeStamp());
```

Comment:
Is this place ok?

Reply:
This gets the timestamp from `TopicConfigs` class. In `TopicConfigs` it is set in two ways: by reading the timestamp of Kafka message (this occurrence is fixed) or by reading a user-provided field (in that case the behaviour should not be changed) - therefore no `* 1000` here (only adding it to all occurances of Kafka's `record.timestamp()`).